src/utils/organization.tsx

```diff
@@ -27,7 +35,15 @@ export const isContributor = (roleHolder: Organization | OrganizationUser | unde
 };
 
 export const getAllNurseries = (organization: Organization): Facility[] => {
-  return getFacilitiesByType(organization, 'Nursery');
+  return getFacilitiesByType(organization, 'Nursery').sort((a, b) => {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1;
+    }
+    return 0;
+  });
```

Comment:
usually repeated code should be extracted out

Reply:
also you can move the sorting inside getFacilitiesByType so we don't need to call it in the calling-code

Reply:
another option is to leave the sorting up to the rendering component

Reply:
oops wrong place